### PR TITLE
fix: replace hidden div examples

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/mdn/kitchensink/index.md [hunk 1]
```diff
@@ -77,20 +77,24 @@ const f = () => {
 
 ### WebAssembly
 
-<pre class="brush: wasm">(func (param i32) (param f32) (local f64)
+```wasm
```
Comment (nschonni) [inline files/en-us/mdn/kitchensink/index.md:80]: This code language was used in other files, so I assume this style isn't needed anymore (in the kitchen sink)

File: files/en-us/web/accessibility/aria/roles/structure_role/index.md [hunk 2]
```diff
@@ -42,15 +42,13 @@ Do not use `role="structure"`. Do use HTML and subclass structure roles.
 - [ARIA: `section` role](/en-US/docs/Web/Accessibility/ARIA/Roles/section_role)
 - [ARIA: `sectionhead` role](/en-US/docs/Web/Accessibility/ARIA/Roles/sectionhead_role)
 
-<div class="hidden">
 <!-- these shouldn't be used so we shouldn't link to them
```
Comment (nschonni) [inline files/en-us/web/accessibility/aria/roles/structure_role/index.md:45]: I'm guessing these could just be deleted, but left them for now

File: files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.md [hunk 3]
```diff
@@ -51,9 +51,16 @@ div {
 
 Both pieces of text have their default black color. The "good" `<div>` has a light purple background, which makes the text easy to read:
 
-<div class="hidden"><h4 id="example1">example1</h4><pre class="brush: html">    &#x3C;div class="good">
-      Good contrast
-    &#x3C;/div></pre><pre class="brush: css">div {
+### Good Contrast
+
+```html
```
Comment (nschonni) [inline files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.md:56]: Could add the `hidden` metadata if the preference is not to have them rendered

**Review by estelle — APPROVED** [review]
Thank you!

**Pull request merged**